tests/testthat/_snaps/badge.md

```diff
@@ -7,3 +7,43 @@
       ! `stage` must be one of "experimental", "stable", "superseded", or "deprecated", not "eperimental".
       i Did you mean "experimental"?
 
+# use_rscloud_badge() handles bad and good input
+
+    Code
+      use_posit_cloud_badge()
+    Condition
+      Error in `is.factor()`:
+      ! argument "url" is missing, with no default
```

Comment:
This is sort of gross. Maybe we should throw in a `check_name(url)`.

Reply:
Actually, R 4.3 automatically does something much better, which is nice.

Reply:
That is nicer - but then the snapshot will be different for different versions. `check_name()` is a good idea